examples/JavaProntoClient/build.gradle

```diff
@@ -5,11 +5,6 @@ apply plugin: 'com.bmuschko.tomcat'
  
 repositories {
     jcenter()

```

Comment:
Since we advise it in the readme, do we want to add a `mavenCentral()` call here?  Does this really work without it (or is mavenCentral effectively optional or subsumed by `jcenter()`?

Reply:
Also, I think `jcenter()` went away when `bintray` did.  So we probably want to remove this as well....

Reply:
I tested the example project and it all worked. Wasn't sure if that was because it's the default option or if jcenter pull it in as well. I'll add it just be make things more clear.

Reply:
Turns out the `jcenter()` is still there, albeit readonly.